gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/package-info.java

```diff
@@ -32,6 +34,21 @@
  * RemoteGcsHelper.forceDelete(storage, bucket, 5, TimeUnit.SECONDS);
  * } </pre>
  *
+ * LocalGcsHelper helps with testing on an in-memory filesystem (this is best for unit tests).
+ * Note that this filesystem isn't a complete implementation. In particular, it is not thread-safe.
+ *
+ * <p>A simple usage example:
+ *
+ * <pre> {@code
```

Comment:
It's safe to omit the {@code} thing here since you don't have any angle brackets.